draft/draft-ietf-ippm-twamp-yang.xml

```diff
@@ -873,9 +873,9 @@ module ietf-twamp {
     This YANG module uses features to indicate which of the four
     logical entities are supported by a TWAMP implementation.";
 
-  revision 2017-10-16 {
+  revision 2018-02-10 {
     description
-      "Revision appearing in draft-ietf-ippm-twamp-yang-05.
+      "Revision appearing in draft-ietf-ippm-twamp-yang-06.

```

Comment:
Since the model is not released, all the description should say is "Initial Revision". After the initial revision is made, one can document the changes made in the draft.

BTW, what we are missing in the draft is textual reference to all the RFCs we refer to in the model. For example, if we cite reference to RFC 5357, we need that in the normative reference of the draft (which we do), but in addition, we need to refer to in the draft before the model. See the ACL model for example.